### PR TITLE
AY-7242_Fix clip_media review source from Hiero.

### PR DESCRIPTION
**Pull request by robin-ynput**
## Changelog Description

When testing Flame and Hiero we realized that the `[Clip media]` option of `Reviewable Source` did not work properly.
Here is the bug fix for Hiero.

## Additional review information
This changes needs to be tested alongside https://github.com/ynput/ayon-core/pull/1036

## Testing notes:
1. Open Hiero and create instance(s) through the `Create Publishable Clip`
2. Set the review source media to `[Clip media]`
image
3. Publish and ensure that the resulting review product is created from the original clip media entire range (untrimmed)

AY-7242

### CODE REVIEW

File: client/ayon_hiero/plugins/publish/collect_plates.py [hunk 1]
```diff
@@ -39,12 +40,45 @@ def process(self, instance):
...
+                _, ext = os.path.splitext(file_url)
+
+                repre = {
+                    "name": ext[1:],
+                    "ext": ext[1:],
+                    "files": file_names,
+                    "stagingDir": os.path.dirname(file_url),
+                    "tags": ["review", "delete"]
+                }
+                instance.data["representations"].append(repre)
```
Comment (jakubjezek001) [inline client/ayon_hiero/plugins/publish/collect_plates.py:74]: We are already having a dedicated plugin for collecting representations from auto clip media reference with proper frame ranges. I wonder whether we should not to distribute any way the `review` tag flagging. Here is the code https://vscode.dev/github/ynput/ayon-core/blob/bugfix/fix_extract_otio_review/client/ayon_core/plugins/publish/collect_otio_subset_resources.py#L207

Comment (robin-ynput) [reply client/ayon_hiero/plugins/publish/collect_plates.py]: Good call, please see https://github.com/ynput/ayon-core/pull/1036.

**Comment by robin-ynput** [comment]
> Awesome @robin-ynput this should be solid. We will also need to add it to Resolve and Flame. I wonder if we could simplify the code here and move the logic to ayon-core as suggested in the comments.

That's a good idea, I have made these changes in `ayon_core`: https://github.com/ynput/ayon-core/pull/1036

**Review by jakubjezek001 — APPROVED** [review]
looking good

**Pull request merged**